apps/music/test/marionette/lib/statusbar.js

```diff
@@ -20,9 +20,10 @@ Statusbar.prototype = {
   },
 
   waitForPlayingIndicatorShown: function(shouldBeShown) {
-    this.client.waitFor(function() {
-      var shown = this.playingIndicator.displayed();
-      return shown === shouldBeShown;
-    }.bind(this));
+    if (shouldBeShown) {
+      this.client.helper.waitForElement(this.playingIndicator);
+    } else {
+      this.client.helper.waitForElementToDisappear(this.playingIndicator);
```

Comment:
waitForElementToDisappear has been around since 2013 [1], but not many people have known about it.

1.) https://github.com/mozilla-b2g/marionette-helper/commit/6513606040e290cf5c1e544a5079113b998e60b7